docs/source/keywords.rst

```diff
@@ -734,6 +734,10 @@ Kokkos will attempt to autodetect the architecture flag at configuration time.
 Intel GPUs
 ~~~~~~~~~~
 
+If the SYCL backend is enabled and no GPU architecture is specified, Kokkos will use Just-In-Time (JIT) compilation.
+There is no restriction to a particular SYCL device type.
+Thus, this is the only option to target a CPU with the SYCL backend (which is experimental, untested, and not optimized for).
+
```

Comment:
Since we already have the explanation of JIT below the table, I would suggest to move/integrate it there and put it into relation with `Kokkos_ARCH_INTEL_GEN`. @crtrott I think it actually is the best place for now

Reply:
I still like to have the footnorte section separated so I just added some references.